packages/popup/src/pages/ConfirmTransaction/components/AddressCard.js

```diff
@@ -78,6 +86,32 @@ AddressDetail.propTypes = {
   isCreateContract: PropTypes.bool,
 }
 
+const useAccountNameByAddress = address => {
+  const {
+    data: {
+      network: {eid: networkId},
+    },
+  } = useCurrentAddress()
+  const {data} = useRPC(
+    [QUERY_ADDRESS, 'useAccountNameByAddress', address, networkId],
```

Comment:
The initial value of address can be undefined .Better like this:
address ? [QUERY_ADDRESS, 'useAccountNameByAddress', address, networkId] :null

Reply:
done